apps/ledger-live-mobile/src/families/polkadot/accountActions.tsx

```diff
@@ -20,11 +21,10 @@ import ChillIcon from "../../icons/VoteNay";
 import { NavigatorName, ScreenName } from "../../const";
 import { ActionButtonEvent } from "../../components/FabActions";
 
-const getActions = ({
-  account,
-}: {
+const getActions = (args: {
   account: Account;
 }): ActionButtonEvent[] | null | undefined => {
+  const account = args.account as PolkadotAccount;
```

Comment:
could just be `const getActions(account: Account)`

Reply:
we try not to impact the logic in this PR